xbmc/addons/InputStream.cpp

```diff
@@ -91,21 +94,22 @@ bool CInputStream::Supports(CFileItem &fileitem)
   }
 
   match = false;
+
   for (auto &path : m_pathList)
   {
     if (path.empty())
       continue;
 
-    if (fileitem.GetPath().compare(0, path.length(), path) == 0)
+    CRegExp r(true, CRegExp::asciiOnly, path.c_str());
```

Comment:
As told before - I have dynamic (1000) different subdomains:
http://123.download.de
http://6x7.download.de
http://abc.download.de

They cannot be configured! But they can with the regexp.